localization/english/dvg_names_l_english.yml

```diff
@@ -34,6 +35,7 @@
  Anguyatutu:0 "Anguyatutu"
  Arami:0 "Arami"
  Chapai:0 "Chapai"
+ Denis:0 "Denis"
```

Comment:
This name already is translated. You can remove this here.

Duplicate localization key. Key 'Denis' is defined in both 'localization/english/dvg_names_l_english.yml' and 'localization/english/names_l_english.yml'.